pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -205,7 +205,12 @@ public enum ControllerGauge implements AbstractMetrics.Gauge {
   PAUSELESS_SEGMENTS_IN_UNRECOVERABLE_ERROR_COUNT("pauselessSegmentsInUnrecoverableErrorCount", false),
 
   // ZK JUTE max buffer size in bytes
-  ZK_JUTE_MAX_BUFFER("zkJuteMaxBuffer", true);
+  ZK_JUTE_MAX_BUFFER("zkJuteMaxBuffer", true),
+
+  // The progress of a certain table rebalance job of a table
+  TABLE_REBALANCE_JOB_ADDING_PROGRESS_PERCENT("percent", false),
+  TABLE_REBALANCE_JOB_DELETING_PROGRESS_PERCENT("percent", false);
```

Comment:
nit: can we call them:
`TABLE_REBALANCE_JOB_SEGMENT_ADDITION_PROGRESS_PERCENTAGE`
`TABLE_REBALANCE_JOB_SEGMENT_DELETETION_PROGRESS_PERCENTAGE`

Just don't use `adding` and `deleting`